components/post_view/post_header/index.js

```diff
@@ -3,15 +3,19 @@
 
 import {connect} from 'react-redux';
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
+import {getUser} from 'mattermost-redux/selectors/entities/users';
 
 import PostHeader from './post_header.jsx';
 
-function mapStateToProps(state) {
+function mapStateToProps(state, ownProps) {
     const config = getConfig(state);
     const enablePostUsernameOverride = config.EnablePostUsernameOverride === 'true';
+    const user = getUser(state, ownProps.post.user_id);
+    const isBot = user ? user.is_bot : false;
```

Comment:
Minor nit. This sort of thing could be 
```suggestion
    const isBot = Boolean(user && user.is_bot);
```

Reply:
I forgot about that. Thanks!